bitnami/jenkins/Chart.yaml

```
@@ -35,4 +35,4 @@ maintainers:
 name: jenkins
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/jenkins
-version: 13.4.6
+version: 13.4.7
```

```suggestion
version: 13.5.0
```

Moving to a new minor version as you are adding new parameters.